src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -673,6 +694,8 @@ def __call__(
         hidden_states = jax.nn.gelu(self.conv2(hidden_states), approximate=False)
 
         embed_positions = self.embed_positions(jnp.arange(self.config.max_source_positions))
+        # freeze the sinusoidal embeddings by stopping the back-prop
```

Comment:
Note to reviewer: by default we freeze the embeddings in Flax, and don't provide an override. See this explanation for detail: https://github.com/huggingface/transformers/pull/26032#issuecomment-1719537609